README.md

```diff
@@ -8,9 +8,10 @@ _Yu-Gi-Oh! Trading Card Game_ and _Official Card Game_. This is the repository f
 to take advantage of new features offered by Discord and improve reliability.
 
 The new instance runs concurrently with the old instance.
-Features implemented here will be gradually rolled out to the live bot. For most documentation, please
-refer to the old [Bastion Classic](https://github.com/AlphaKretin/bastion-bot) repository. For
-Slash Command documentation, look in [`docs/commands`](/docs/commands).
+Features implemented here will be gradually rolled out to the live bot.
+New `<>` card search documentation is [HERE](/docs/card-search.md).
```

Comment:
the allcaps is a bit out of place. Markdown provides a lot of more natural ways to draw attention. IDK whats proven to best get attention though

Reply:
Ah, I can bold it instead?

Reply:
Yeah, that sounds good